packages/ibm-products/src/components/AddSelect/AddSelectRow.js

```diff
@@ -158,19 +158,34 @@ export let AddSelectRow = ({
               />
               {hasModifiers && (
                 <Layer>
-                  <Dropdown
-                    titleText={modifiers.label}
-                    id={`add-select-modifier-${item.id}`}
-                    type="inline"
-                    items={modifiers.options}
-                    label={modifiers.label}
-                    disabled={!isSelected()}
-                    className={`${blockClass}-dropdown`}
-                    initialSelectedItem={item[modifiers.id]}
-                    onChange={({ selectedItem }) =>
-                      modifierHandler(item.id, selectedItem)
-                    }
-                  />
+                  {modifiers?.multiSelect ? (
+                    <MultiSelect
+                      titleText={modifiers.title}
```

Comment:
if the props here and in `Dropdown` are all the same we should just have a props variable and pass it via that instead of adding redundant code. something like...
```
const sharedProps = {
  titleText: modifiers.title
  ...
};
...
<MultiSelect {...sharedProps} /> : <Dropdown {...sharedProps} />
```